### ISSUE TITLE
Case sensitive setting, wrong work on windows 

### ISSUE THREAD

**Issue by Leeron2010** [zertosh/eslint-plugin-dependencies#11]
Hello.

**Comment by alundiak** [comment]
On my Windows, I have code:
```
require('./views/closeAssignment/AssignmentCloseRatingLayout')
```

which is valid filename location, path and case. 
And when I fake setup this code:

```
require('./views/CloseAssignment/assignmentcloseratinglayout')
```
then `"dependencies/case-sensitive": 2,` setup doesn't throw an error.

Is this the same issue as @Leeron2010 described?
